### PR TITLE
Handle "merged" pull request

### PR DESCRIPTION
**Pull request by SantiMA10**
Related to #26 

> Draft until #29 

This PR adds support to send pull request "merged" events to StreamLabs

### CODE REVIEW

File: src/routes/github/index.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+				payload.pull_request.merged
+			) {
+				const {
+					sender: { login: senderLogin },
+				} = payload;
+
+				await axios.post(config.STREAMLABS_ENDPOINT, {
+					access_token: config.STREAMLABS_TOKEN,
+					type: 'follow',
+					message: `The pull request from *${senderLogin}* just merged *${repositoryFullName}*`,
```
Comment (orestes) [inline src/routes/github/index.js:63]: Maybe "has been merged into"?

**Comment by orestes** [comment]
Could you please rebase this PR onto master @SantiMA10

**Comment by orestes** [comment]
@SantiMA10 can you rebase and review conflicts @SantiMA10 ?

**Comment by SantiMA10** [comment]
@orestes done!

**Pull request merged**